src/Middleware/Diagnostics/test/FunctionalTests/Diagnostics.FunctionalTests.csproj

```diff
@@ -5,6 +5,9 @@
     <SignAssembly>false</SignAssembly>
     <AssemblyName>Diagnostics.FunctionalTests</AssemblyName>
     <TestDependsOnMssql>true</TestDependsOnMssql>
+
+    <!-- https://github.com/dotnet/aspnetcore/issues/38819 LocalDb sometimes hangs on win11 helix queue -->
+    <BuildHelixPayload>false</BuildHelixPayload>
```

Comment:
> @dougbu had attempted to port this back to 6.0 yesterday: https://github.com/dotnet/aspnetcore/pull/38814#issuecomment-1032831047. I'm not super sure of the background on this though.

The background is we need this addition in release/6.0 to get PRs passing because LocalDB isn't reliable where we need it to be reliable.

The un-quarantining is also fine as long as the tests have been just as reliable in this branch. If the test or test infrastructure isn't up to the task, we should backport _only_ this change. I'm not familiar enough w/ the tests to know either way and am asking the gathered masses (as I would have if the backport bot could work)&hellip;

Reply:
/btw this lack is also holding up #40073

Reply:
> I'm not familiar enough w/ the tests to know either way and am asking the gathered masses (as I would have if the backport bot could work)…

Checked the analytics and the tests have been passing at 100% in this branch too 🎉